src/components/form/components/errors/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconAlertDiamondFill16 } from '@hashicorp/flight-icons/svg-react/alert-diamond-fill-16'
+import { ErrorsProps } from './types'
+import s from './errors.module.css'
+import classNames from 'classnames'
+
+const Errors = ({ className, messages = [] }: ErrorsProps) => {
+	const classes = classNames(s.root, className)
+
+	return (
+		<div className={classes}>
+			<IconAlertDiamondFill16 className={s.icon} />
+			<ul className={s.list}>
+				{messages.map((error: string) => (
+					// eslint-disable-next-line react/jsx-key
+					<li className={s.message}>{error}</li>
```

Comment:
suggestion: instead of disabling this, could we use the message as a key?

Reply:
Oh, oops! I meant to make that the array index, not totally disable 😆

Reply:
✅ Fixed in: https://github.com/hashicorp/dev-portal/pull/872/commits/e7bc40697947acf57081819bc027eb0dbc522220